example/script-update/go.mod

```diff
@@ -0,0 +1,42 @@
+module script-update
```

Comment:
it looks like go.sum file missing. can you run `go mod tidy` please?

Reply:
done